bliss/conf/base_config.yaml

```diff
@@ -143,17 +143,17 @@ variational_factors:
 metrics:
     detection_performance:
         _target_: bliss.encoder.metrics.DetectionPerformance
-        base_nmgy_bin_cutoffs: [200, 400, 600, 800, 1000]
-        report_bin_unit: njymag
+        base_njy_bin_cutoffs: [200, 400, 600, 800, 1000]
```

Comment:
Seems better to always specify cutoffs in magnitude, and to always report them in the same units